core/src/main/java/org/elasticsearch/index/seqno/LocalCheckpointService.java

```diff
@@ -32,33 +33,41 @@
  */
 public class LocalCheckpointService extends AbstractIndexShardComponent {
 
+    public static String MAX_SEQ_NO = "max_seq_no";
```

Comment:
I pushed a5b99c65259b933c4b379e29e160685ee427eda9 and 6e8f11ae3da78c4b82cd267c1daba3951b462eee.